staking/handler_test.go

```diff
@@ -148,12 +148,12 @@ func (suite *HandlerTestSuite) TestHandleMsgValidatorUpdate() {
 
 	ValFrmID, ok := keeper.GetValidatorFromValID(ctx, oldSigner.ID)
 	require.True(t, ok, "new signer should be found, got %v", ok)
```

Comment:
when you call handler, state won't be changed. So it will be old signer. 
msg is misleading.